### PR TITLE
Click event processing

### PR DESCRIPTION
**Pull request by pgetta**

### CODE REVIEW

File: apps/search-poc/src/index.tsx [hunk 1]
```diff
@@ -2,18 +2,21 @@ import * as React from 'react';
 import * as serviceWorker from './serviceWorker';
 import { render } from 'react-dom';
 import { Switch, Route } from 'react-router';
-import { BrowserRouter as Router } from 'react-router-dom';
```
Comment (kenjinp) [inline apps/search-poc/src/index.tsx]: I guess technically we don't need the qRouter` anymore and can go back to the BrowserRouter, which will save us having to created the `src/history.ts` file

File: apps/search-poc/src/containers/WorkspaceList.tsx [hunk 2]
```diff
@@ -19,8 +22,9 @@ type WorkspaceConfig = {
 const WorkspaceListContainer: React.FunctionComponent<{
   workspaceConfig: WorkspaceConfig[];
   onWorkspaceSelected: (workspaceId: string) => void;
-  history: History;
-}> = ({ workspaceConfig, onWorkspaceSelected, history, children }) => {
+  handleClick: (params: HandleClickParams) => void;
+}> = ({ workspaceConfig, onWorkspaceSelected, handleClick }) => {
```
Comment (kenjinp) [inline apps/search-poc/src/containers/WorkspaceList.tsx]: children is left undefined after merge I guess

> ./src/containers/WorkspaceList.tsx

  > Line 73:  'children' is not defined  no-undef

Comment (pgetta) [reply apps/search-poc/src/containers/WorkspaceList.tsx]: Fixed

File: apps/search-poc/src/index.tsx (hunk 1 above)
Comment (pgetta) [reply apps/search-poc/src/index.tsx]: I've fixed that as well.

File: apps/search-poc/package.json [hunk 3]
```diff
@@ -43,40 +25,61 @@
   "eslintConfig": {
     "extends": "react-app"
   },
+  "jest": {
```
Comment (julienmachon) [inline apps/search-poc/package.json]: Have you rebased? I don't think you should have this as a change as it's been merged in a previous PR already?

Comment (pgetta) [reply apps/search-poc/package.json]: Yeah, I've removed those changes.

**Pull request merged**